### DISCUSSION TITLE
请问这个主题如何更新呢？

### DISCUSSION THREAD

**Discussion by qhmg** [Tokinx/Wing#74] [问答]
第一个问题：是每次在github上下载压缩包上传到wordpress更新？还是用wordpress自带的更新？
第二个问题：可是有个问题，我这github上下载的压缩包上传上去安装的主题，与系统自带更新提醒的版本不一致，而且系统提醒的版本小了我现在安装的版本好几个版号，不知道怎么回事，越更新版本号越高才对呀。
<img width="1372" alt="image" src="https://user-images.githubusercontent.com/118754087/203099187-62eccea8-754f-48ad-a4db-f9d441ddede7.png">

**Comment by Tokinx** [comment]
应该是要在主题设置里开启接收预览版更新选项。否则会以为你要接收稳定版。

不过实际上主题没有稳定版，这个下次更新我会取消这个逻辑判断

**Comment by Tokinx** [comment]
另外，主题自带更新，不需要手动更新，新版本会推送到你WP后台，更新即可。
